src/PageNotFound/PageNotfound.tsx

```diff
@@ -0,0 +1,18 @@
+import { Container, Grid } from '@material-ui/core'
+import React from 'react'
+import  PNFImage from '../Media/PageNotFoundImage.jpg'
+
+interface Props {}
+
+function PageNotFound() {
+	return (
+		<Grid container alignItems="center" justify="center">
+			<Grid item>
+				<h3>Page not found. How did you even get here?</h3>
```

Comment:
Instead of the h3 HTML tag, we should use Typography here; I'll pull down the branch and make some minor updates to address this